contrib/inventory/netbox.py

```diff
@@ -0,0 +1,378 @@
+#!/usr/bin/env python
+
+# (c) 2017, Ahmed AbouZaid <http://aabouzaid.com/>
+#
+# This file is part of Ansible,
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible. If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import sys
+import yaml
+import argparse
+import requests
```

Comment:
Add try-excepts for the `requests` module as well.

Reply:
Fixed.